lib/postgres_ext/arel/5.0/visitors/postgresql.rb

```diff
@@ -0,0 +1,61 @@
+require 'arel/visitors/postgresql'
```

Comment:
At the moment only `visit_Arel_Nodes_Contains` is different, but I'm still working on modifying it.